pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -204,6 +224,7 @@ func (s *Service) createInstance(machine *actuators.MachineScope, bootstrapToken
 	if err != nil {
 		return nil, err
 	}
+	machine.ClusterControlPlaneStatus().ControlPlaneNodeCount = controlPlaneNodeCount
```

Comment:
Same as the other comment, if the increments/decrements are functions we wouldn't need to do this assignment.

Reply:
Outdated